src/brsTypes/components/RoAssociativeArray.ts

```diff
@@ -17,13 +17,15 @@ export interface AAMember {
 export class RoAssociativeArray extends BrsComponent implements BrsValue, BrsIterable {
     readonly kind = ValueKind.Object;
     elements = new Map<string, BrsType>();
+    /** keyMap - maps lowercased key to original one which used in this.elements to store value
+     * Main benefit of it is fast case insensitive access
+     */
```

Comment:
nit: a little syntax
```suggestion
    /** Maps lowercased element name to original name used in this.elements.
     * Main benefit of it is fast, case-insensitive access.
     */
```